price-feeder/oracle/util.go

```diff
@@ -45,3 +45,78 @@ func ComputeVWAP(prices map[string]map[string]provider.TickerPrice) (map[string]
 
 	return vwap, nil
 }
+
+// FilterDeviations find the standard deviation of the price
+// of an asset, and filter out any providers that are not within
+// two deviations of the mean.
+func FilterDeviations(
+	prices map[string]map[string]provider.TickerPrice) (
+	map[string]map[string]provider.TickerPrice, error,
+) {
```

Comment:
```suggestion
) {
	if len(prices) < 3 {
		return prices
	}
```
Automatically return early if there aren't enough providers to support the math

Reply:
Good idea, did this in ee0582dd15e4a834dbbfdf50398944d50325067d if you're interested - we'll likely never hit this bit once I implement the three-providers requirement in config validation, except for mocked rates, which we'll have to allow for testnet